src/containers/Topbar/Topbar.js

```diff
@@ -85,12 +93,48 @@ class Topbar extends React.PureComponent {
     this.handleCancel = this.handleCancel.bind(this);
     this.onAddressInputChange = this.onAddressInputChange.bind(this);
     this.onAddressDropdownClick = this.onAddressDropdownClick.bind(this);
-    this.getSelectedAddressObject = this.getSelectedAddressObject.bind(this);
     this.onCopyClicked = this.onCopyClicked.bind(this);
   }
 
   componentWillMount() {
-    this.props.listUnspent();
+    const { onGetBlockCount, listUnspent } = this.props;
+
+    (function startPoll() {
+      onGetBlockCount();
+      listUnspent();
+      setTimeout(startPoll, POOL_INTERVAL);
+    }());
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const {
+      onGetBlockCount, getBotBalance, selectedWalletAddress,
+    } = this.props;
+
+
```

Comment:
👾

Reply:
I just realized what this crab means.. have been ignoring this icon for previous PRs...

Reply:
haha its a space invader from the old arcade game